src/logging/request-logger.js

```diff
@@ -15,8 +15,8 @@ module.exports = logger => requestLogger(request, (request, instance) => {
   obfuscate(request, instance);
 
   if (request.type === 'response') {
-    return logger.debug({ request }, `Received response for request ${request.id}`);
+    return logger.debug({ request }, `Received response`);
   }
 
-  return logger.debug({ request }, `Making request ${request.id} to ${request.method} ${request.uri}`);
+  return logger.debug({ request }, `Sending request ${request.method} to ${request.uri}`);
```

Comment:
I would remove logging of the `uri` to prevent leaking api keys that are embedded within it.

```suggestion
  return logger.debug({ request }, `Sending ${request.method} request`);
```

Reply:
Or 
```suggestion
  return logger.debug({ request }, 'Sending request');
```